functions/index.js

```diff
@@ -60,6 +60,53 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
     });
 });
 
+exports.createNewSprint = functions.https.onRequest((request, response) => {
+    cors(request, response, () => {
+        console.log(request);
+        var title = request.body.data.Title;
+        var des = request.body.data.Description;
+        var startDate = request.body.data.StartDate;
+        var endDate = request.body.data.EndDate;
+        var status = request.body.data.Status;
+        var totalDevelopmentTask = request.body.data.development;
+        var totalBusinessTask = request.body.data.business;
+        var totalMarketingTask = request.body.data.marketing;
+
+
+        console.log(title);
+        console.log(des);
+        console.log(endDate);
+        console.log(startDate);
+        console.log(totalDevelopmentTask);
+        console.log(totalBusinessTask);
+        console.log(totalMarketingTask);
+        console.log(SprintID);
```

Comment:
var sprintid is not yet created and the method must be created to get the sprintId.

The sprintId can be taken from below given address in db.

collection: "Main"
document: "RawDetails"
field: "lastSprintId"

Once the lastSprintId is fetched, New sprintId will be created as given bellow:

newSprintId = lastSprintId+1
sprintId = "S" + newSprintId

and also the value of lastSprintId in same location must be updated with newSprintId.

And then the following location shall be updated.